hal/halarm/generic/reboot.c

```diff
@@ -9,35 +9,41 @@
 /* INCLUDES ******************************************************************/
 
 #include <hal.h>
-#define NDEBUG
-#include <debug.h>
```

Comment:
Question, no request for change: Here you do remove the debug include. I do like that. But then later the source calls stuff like DbgPrint(). Isn't that a problem? Or where does DbgPrint() come from? Thx for explanation in advance.

Reply:
DbgPrint is standard NT function -> it's in the usual WDK/DDK/XDK/NDK headers. reactos' own debug.h is for the ROS-specific debug wrapper macros DPRINT(1) and ERR/WARN/TRACE .